catala/dormir/dormir.md

```diff
@@ -1,2 +1,7 @@
 Et quedes dormint com un suro. A les 14 hores
-et desperta ta mare amb un plat d'arròs al forn.
\ No newline at end of file
+et desperta ta mare amb un plat d'arròs al forn.
+
+Veus que surt fum del plat. Què fas?
+
+[Tasto una mica](tastar/tastar_arros.md)
+[Pregunto a la mare per què surt fum del plat](preguntar/preguntar_fum.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.